src/pcre2_compile.c

```diff
@@ -9812,17 +9812,17 @@ PCRE2_UCHAR *cworkspace = (PCRE2_UCHAR *)c16workspace;
 
 /* -------------- Check arguments and set up the pattern ----------------- */
 
-/* There must be error code and offset pointers. */
+/* There must be an error offset pointer. */
 
-if (errorptr == NULL || erroroffset == NULL) return NULL;
-*errorptr = ERR0;
+if (erroroffset == NULL) return NULL;
+if (errorptr) *errorptr = ERR0;
```

Comment:
noticed this is not documented, could it be dropped (meaning errorptr is indeterminate unless return is NULL)?
eitherway it should be documente IMHO, which I am planning to do in a followup commit.